includes/sanitizers/class-amp-allowed-tags-generated.php

```diff
@@ -1657,9 +1657,7 @@ class AMP_Allowed_Tags_Generated {
 						'value' => '',
 					),
 					'sandbox' => array(),
-					'scrolling' => array(
-						'value_regex' => 'auto|yes|no',
-					),
+
```

Comment:
Why is this change here? Any changes to this generated file should be done via [`amphtml-update.sh`](https://github.com/Automattic/amp-wp/blob/develop/bin/amphtml-update.sh)